x/wasm/Governance.md

```diff
@@ -0,0 +1,110 @@
+# Governance
```

Comment:
This file should be rewritten. so I think it would be better to remove it for now.

Reply:
Ok, removed it.